app/frontend/src/router/index.js

```diff
@@ -17,8 +17,20 @@ const router = new VueRouter({
     },
     {
       path: '/admin',
-      name: 'Admin',
       component: () => import(/* webpackChunkName: "admin" */ '@/views/Admin.vue'),
+      children: [
+        {
+          path: '',
+          name: 'Admin',
+          component: () => import(/* webpackChunkName: "submission-table" */ '@/views/admin/Root.vue')
+        },
+        {
+          path: '/submission/:ipcPlanId',
```

Comment:
i was wondering if the submission page should be at the url  '/admin/submission/xyz..' 
(i actually dont see why it's not working that way as you have it. i better learn more about child routes)

Reply:
That path should actually be `submission/:ipcPlanId` without the leading slash as that makes it absolute. Thanks for the catch. With child paths, they should be relatively defined as their parent(s) will be able to define the absolute path base. You don't need to put in /admin again because the parent defines it.

Reply:
nm my comment was after recent fix